gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -466,9 +466,8 @@ public synchronized void scheduleJob(Properties jobProps, JobListener jobListene
   @Override
   protected void logNewlyScheduledJob(JobDetail job, Trigger trigger) {
     Properties jobProps = (Properties) job.getJobDataMap().get(PROPERTIES_KEY);
-    log.info(jobSchedulerTracePrefixBuilder(jobProps) + "nextTriggerTime: {} localTZNextTriggerTime:{} - Job newly "
-            + "scheduled", utcDateAsUTCEpochMillis(trigger.getNextFireTime()),
-        systemDefaultZoneDateAsUTCEpochMillis(trigger.getNextFireTime()));
+    log.info(jobSchedulerTracePrefixBuilder(jobProps) + "nextTriggerTime (in UTC): {} - Job newly "
```

Comment:
rather than sprinkling " (in UTC)" so many places, what if we modified `jobSchedulerTracePrefixBuilder` so it read something like:
```
"Scheduler trigger tracing (in epoch-ms UTC): [flowName:..."
```
?

(so just once in the log line's prefix)